collect_app/src/main/java/org/odk/collect/android/preferences/AggregatePreferencesActivity.java

```diff
@@ -55,6 +76,7 @@ protected void onPause() {
         }
     }
 
+    @TargetApi(Build.VERSION_CODES.HONEYCOMB)
```

Comment:
This is not necessarily a problem but I do want to understand it. I'm hoping we can address #293 before the next release anyway.